src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.po.ts

```diff
@@ -1,5 +1,157 @@
+import { $, browser, by, element } from 'protractor';
+// import { protractor } from 'protractor/built/ptor';
+import { Helper } from '../helper.po';
 import { PageHelper } from '../page-helper.po';
 
-export class BucketsPage extends PageHelper {
-  pages = { index: '/#/rgw/bucket' };
+export class BucketsPageHelper extends PageHelper {
+  static pages = {
+    index: '/#/rgw/bucket',
+    create: '/#/rgw/bucket/create'
+  };
+
+  static navigateTo(page = null) {
+    return browser.get(this.pages[page || 'index']);
+  }
+
+  static create(name, owner) {
+    this.navigateTo('create');
+    expect(PageHelper.getTitleText()).toBe('Create Bucket');
+
+    // Need the string '[value="<owner>"]' to find the owner in dropdown menu
+    const str1 = '[value="';
+    const str2 = '"]';
+    const str3 = str1.concat(owner);
+    const getOwnerStr = str3.concat(str2);
```

Comment:
You only need 1 const to create the desired string by using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
```
const getOwnerStr = `[value="${new_owner}"]`;
```